mypy_plugins/sympy.py

```diff
@@ -0,0 +1,59 @@
+from mypy.plugin import Plugin
+from mypy.plugins.common import add_attribute_to_class
+from mypy.types import NoneType, UnionType
+
+
+class SympyPlugin(Plugin):
+    def get_base_class_hook(self, fullname: str):
+        if fullname == "sympy.core.basic.Basic":
+            return add_assumptions
+        return None
+
+
+def add_assumptions(ctx) -> None:
+    # Generated by list(sys.modules['sympy.core.assumptions']._assume_defined)
```

Comment:
Given that our .pyi are already generated at build time and sympy is a build dependency (is it?) 
Should we dynamically generate this at build time to avoid divergence?

Reply:
meh